Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:24.04 AS seiscomp
+ADD https://github.com/GeoscienceAustralia/ga-wphase/releases/download/v0.3.2/seiscomp-6.5.1-ubuntu24.04-x86_64.tar.gz /opt/
+RUN cd /opt && tar xfz seiscomp*.tar.gz
+
+FROM ubuntu:24.04 AS wphase
+RUN apt-get update &&  \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+        build-essential \
+        ca-certificates \
+        gfortran \
+        libboost-filesystem1.83.0 \
+        libboost-iostreams1.83.0 \
+        libboost-program-options1.83.0 \
+        libboost-regex1.83.0 \
+        libboost-system1.83.0 \
+        libboost-thread1.83.0 \
+        libmariadb3 \
+        libmysqlclient21 \
+        libncurses6 \
+        libpq5 \
+        libproj-dev \
+        libssl3 \
+        libxml2 \
+        python-is-python3 \
+        python3-dev \
+    && apt-get clean autoclean -y && rm -rf /var/lib/apt/lists/*
+
+COPY --from=ghcr.io/astral-sh/uv:0.5.6 /uv /uvx /bin/
```

Comment:
I don't know if it's intentional or not but there's the uv install here but also the uv mount in the run commands below